src/components/Books.js

```diff
@@ -12,23 +11,18 @@ const Container = styled.div`
 `;
 
 function Books() {
-  const books = useSelector(selectBooks);
+  const books = useSelector((state) => state.books);
   return (
     <Container>
-      {books.length ? (
-        books.map((book) => (
-          <CardBook
-            key={book.id}
-            id={book.id}
-            title={book.title}
-            author="Shedrack Sunday"
-            category={book.category}
-          />
-        ))
-      ) : (
-        <h1>Add new Book!</h1>
-      )}
-
+      {books.map((book) => (
+        <CardBook
+          key={book.item_id}
+          id={book.item_id}
+          name={book.title}
+          author="Shedrack Sunday"
+          category={books.category}
```

Comment:
It says, The prop `category` is marked as required in `CardBook`, but its value is `undefined`. Kindly fix this error.

<img width="811" alt="Screenshot 2022-11-12 at 1 36 58 PM" src="https://user-images.githubusercontent.com/83048208/201466184-260668c1-6045-42b2-a6a6-f816c02238ba.png">